projects/git-lfs.github.com/package.yml

```diff
@@ -0,0 +1,46 @@
+distributable:
+  url: https://github.com/git-lfs/git-lfs/releases/download/v{{version}}/git-lfs-v{{version}}.tar.gz
+  strip-components: 1
+
+versions:
+  github: git-lfs/git-lfs
+
+dependencies:
+  git-scm.org: '*'
+  
+build:
+  dependencies:
+    tea.xyz/gx/cc: c99
+    tea.xyz/gx/make: '*'
+    gnu.org/wget: '*'
+  
+  script:
+    - wget $DOWNLOAD_URL
+    - $UNARCHIVE
```

Comment:
Are we unable to use https://github.com/git-lfs/git-lfs#from-source to build it? if so, it should have the 
```yaml
warnings:
  - vendored
```
block at the top level.

Reply:
This is the only way to avoid the "error: git-lfs died on signal 11" error when testing. I couldn't think of a better solution :)